docs/testing.md

```diff
@@ -9,6 +9,8 @@ To define a test you need to call `Deno.test` with a name and function to be
 tested. There are two styles you can use.
 
 ```ts
+import { assertEquals } from "https://deno.land/std@0.82.0/testing/asserts.ts";
```

Comment:
replace version number with `$STD_VERSION` so it's automatically filled with proper value in rendered page

Reply:
Resolved